tabu/tabu_search.pyx

```diff
@@ -26,8 +27,16 @@ cdef class TabuSearch:
     """Wraps the class `TabuSearch` from `tabu_search.cc`."""
     cdef tabu.TabuSearch *c_tabu
 
-    def __cinit__(self, object Q, object initSol, int tenure, int timeout, int numRestarts, object seed=None) :
+    def __cinit__(self, 
+                  object Q, 
+                  object initSol, 
+                  int tenure, 
+                  int timeout, 
+                  int numRestarts, 
+                  object seed=None, 
+                  object energyThreshold=None) :
         cdef unsigned int _seed = time(NULL) if seed is None else seed
+        cdef double _energyThreshold = -numeric_limits[double].max() if energyThreshold is None else energyThreshold
```

Comment:
How about using `-inf` instead? Functionally is the same, but conceptually nicer.

Reply:
Done!